w3-account.md

```diff
@@ -76,68 +108,78 @@ Alice -->> App: alice@web.mail
 App->>Space:🔑 Create namespace
 
 Space ->> Email: 🎫 Delegate capabilities
-note over Space,Email:can:*<br/>with: ucan:*
+note over Space,Email:can:*<br/>with: did:key:z7Gf…xSpace
 ```
 
-> During the first run, a new namespace is generated and complete authority is delegated to the user account. Illustration leaves out steps in which application attempts to claim capabilities before deciding to create a new space.
+> On first run, application creates a new namespace and delegates full authority to the user account. For the simplicity we omit steps where application first requests access from an account before deciding to create new space.
+
+### Delegating capabilities
 
-Any user CAN delegate or re-delegate capabilities to their peer by delegating to their [account] identified by a [`did:mailto`] _(which can be derived from their email address)_. It's worth noting that no setup is required from the delegate until they decide to invoke the delegated capability.
+[Account] CAN authorize user agents by re-delegating a set of capabilities.
 
 ```mermaid
 sequenceDiagram
 participant Alice as 👩‍💻<br/><br/>did:key:z6Mkk…sxALi
 participant Email as 📫 bob@gmail.com
 participant Bob as 👨🏽‍💻 <br/><br/>did:key:z6Mkr…jnz2z
 
-
 Alice ->> Email: 🎫 Delegate capabilities
-note over Alice,Bob:<br/>with: space://z6Mkr…jnz2z<br/>can:store/add
+note over Alice,Bob:<br/>with: did:key:z6Mkk…sxALi<br/>can:store/add
 Email ->> Bob: 🎫 Delegate capabilities
 ```
 
-> **Alice** delegates the `store/add` capability to **Bob**, who later creates an agent and re-delegates the capability to it.
+> **Alice** delegates the `store/add` capability to **Bob**s [account]. Later **Bob**s user agent gets re-delegated the capability from the [account].
 
-### Delegating capabilities
+### Authorization
 
-Delegating capabilities from an [account] identified by a [`did:mailto`] to an [agent] is less straightforward because the signing key is not self-evident from the delegation.
+Delegations issued by [`did:key`] principal are authorized by signing payload with private key. Delegations issued by a [`did:mailto`] principal are not authorized by signing over payload as there is no public key associated with [`did:mailto`] to sign it with. Instead, such delegations are authorized through an interactive email flow in which [account] holder is able to review and approve requested authorization through an action.
 
-To address this issue, we propose an extension to the [UCAN] specification that allows signing and verification of delegations issued by [`did:mailto`] without requiring a [`did:mailto`] key resolution.
+Since there is no public key to sign the [UCAN] payload define an extension to the [UCAN] specification introducing two new signature types that can be used in delegations issued by [`did:mailto`] principals.
```

Comment:
```suggestion
Since there is no private key to sign the [UCAN] payload, we define an extension to the [UCAN] specification that introduces two new signature types that can be used in delegations issued by [`did:mailto`] principals.
```